frontend/src/utils/relayer.ts

```diff
@@ -2,20 +2,25 @@
  * @notice Class for managing relayed withdrawal transactions
  */
 
-import { JsonRpcProvider } from 'src/utils/ethers';
+import { JsonRpcProvider, BigNumber, parseUnits } from 'src/utils/ethers';
 import {
   ConfirmedITXStatusResponse,
   FeeEstimateResponse,
   ITXStatusResponse,
   Provider,
-  TokenInfo,
+  TokenInfoWithMinSendAmount,
   RelayResponse,
   TokenListResponse,
   WithdrawalInputs,
 } from 'components/models';
 
 export class ITXRelayer {
```

Comment:
Since we also use OZ now, this is no longer ITX specific so we should probably rename this class this to `UmbraRelayer` or or similar